utils/check_availability.js

```diff
@@ -16,11 +16,11 @@
  */
 
 const assert = require('assert');
-const playwright = require('..').chromium;
+const browserFetcher = require('../lib/server/browserFetcher.js');
 const https = require('https');
 const SUPPORTER_PLATFORMS = ['linux', 'mac', 'win32', 'win64'];
 
-const fetchers = SUPPORTER_PLATFORMS.map(platform => playwright._createBrowserFetcher({platform}));
+const fetcherOptions = SUPPORTER_PLATFORMS.map(platform => ({platform: platform === 'mac' ? 'mac10.15' : platform, browser: 'chromium'}));
```

Comment:
I have a mac10.14 and would like to check availability :) Or did I misunderstand this?

Reply:
this script currently only works for Chromium, and chromium is the same binary for 10.14 and 10.15!